assets/src/application/services/toaster/style.scss

```diff
@@ -35,6 +36,10 @@
 				}
 			}
 		}
+
+		.Toastify__close-button {
+			color: var(--ee-default-text-color);
```

Comment:
fixed overridden color of success close btn:

![image](https://user-images.githubusercontent.com/1477453/81835013-6b632680-954a-11ea-900a-956d2190fe49.png)